templates/vue-demo-store/app/components/account/AccountChangeEmail.vue

```diff
@@ -107,7 +107,7 @@ onBeforeMount(() => {
             v-if="$v.email.$error"
             class="text-red-600 focus:ring-primary border-secondary-300 rounded"
           >
-            {{ $v.email.$errors[0].$message }}
+            {{ $v.email.$errors[0]?.$message }}
```

Comment:
it should be done on `if` level:
`v-if="$v.email.$error && $v.email.$errors[0]?.$message"`
or
`v-if="$v.email.$error && $v.email.$errors.length > 0"`

Because now it looks like `span` can be displayed without content

Reply:
applied